Android/app/src/main/res/values_fr/strings.xml

```diff
@@ -0,0 +1,88 @@
+<!--
```

Comment:
The directory path needs to be fixed so Android Studio finds this file. Please change values_fr to values-fr.